package.json

```diff
@@ -77,8 +77,9 @@
         "socket.io": "^2.0.4",
         "strip-ansi": "^3.0.0",
         "style-loader": "^0.19.0",
+        "ts-loader": "^2.3.7",
```

Comment:
@WatsonCIQ consider swapped ts-loader with awesome-typescript-loader for consistency with the finsemble core.